app/src/main/java/com/macbitsgoa/ard/utils/HandleFirebaseMessages.java

```diff
@@ -0,0 +1,66 @@
+package com.macbitsgoa.ard.utils;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.util.Log;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Map;
+
+public class HandleFirebaseMessages extends FirebaseMessagingService {
+
+    public static Bitmap getBitmapFromURL(String src) {
+        try {
```

Comment:
I just need to extract bitmap , so , i don't think using Glide will make any difference .